qiskit_ibm_runtime/options/estimator_options.py

```diff
@@ -12,101 +12,114 @@
 
 """Estimator options."""
 
-from typing import Union, Literal
+from typing import Union
 
-from pydantic import Field, field_validator
+from pydantic import Field
 
 from .utils import (
     Dict,
     Unset,
     UnsetType,
-    skip_unset_validation,
 )
 from .execution_options import ExecutionOptionsV2
 from .resilience_options import ResilienceOptionsV2
 from .twirling_options import TwirlingOptions
+from .dynamical_decoupling_options import DynamicalDecouplingOptions
 from .options import OptionsV2
-from .utils import primitive_dataclass
+from .utils import primitive_dataclass, make_constraint_validator
 
-DDSequenceType = Literal["XX", "XpXm", "XY4"]
 MAX_RESILIENCE_LEVEL: int = 2
 MAX_OPTIMIZATION_LEVEL: int = 1
 
 
 @primitive_dataclass
 class EstimatorOptions(OptionsV2):
-    """Options for EstimatorV2.
+    """Options for V2 Estimator.
 
     Args:
+        default_precision: The default precision to use for any PUB or ``run()``
+            call that does not specify one.
+            Each estimator pub can specify its own precision. If the ``run()`` method
+            is given a precision, then that value is used for all PUBs in the ``run()``
+            call that do not specify their own. Default: 0.02.
```

Comment:
The default is actually  0.015625, which is a weird number, because its 1 / sqrt(4096), since 4096 was the previous default shots.